alcs-api/src/application/application.service.ts

```diff
@@ -199,7 +199,7 @@ export class ApplicationService {
     return application;
   }
 
-  async getAllApplicationsWithIncompleteSubtasks(subtaskType: string) {
+  async getBySubtaskType(subtaskType: string) {
```

Comment:
It still uses the incomplete filter, I think it should be reflected somehow

Reply:
Renamed